Sprint-1/implement/max.test.js

```diff
@@ -1,43 +1,73 @@
-/* Find the maximum element of an array of numbers
-
-In this kata, you will need to implement a function that find the largest numerical element of an array.
-
-E.g. max([30, 50, 10, 40]), target output: 50
-E.g. max(['hey', 10, 'hi', 60, 10]), target output: 60 (sum ignores any non-numerical elements)
-
-You should implement this function in max.js, and add tests for it in this file.
-
-We have set things up already so that this file can see your function from the other file.
-*/
-
-const findMax = require("./max.js");
-
-// Given an empty array
-// When passed to the max function
-// Then it should return -Infinity
-// Delete this test.todo and replace it with a test.
-test.todo("given an empty array, returns -Infinity");
-
-// Given an array with one number
-// When passed to the max function
-// Then it should return that number
-
-// Given an array with both positive and negative numbers
-// When passed to the max function
-// Then it should return the largest number overall
-
-// Given an array with just negative numbers
-// When passed to the max function
-// Then it should return the closest one to zero
-
-// Given an array with decimal numbers
-// When passed to the max function
-// Then it should return the largest decimal number
-
-// Given an array with non-number values
-// When passed to the max function
-// Then it should return the max and ignore non-numeric values
-
-// Given an array with only non-number values
-// When passed to the max function
-// Then it should return the least surprising value given how it behaves for all other inputs
+/* Find the maximum element of an array of numbers
+
+In this kata, you will need to implement a function that find the largest numerical element of an array.
+
+E.g. max([30, 50, 10, 40]), target output: 50
+E.g. max(['hey', 10, 'hi', 60, 10]), target output: 60 (sum ignores any non-numerical elements)
+
+You should implement this function in max.js, and add tests for it in this file.
+
+We have set things up already so that this file can see your function from the other file.
+*/
+
+const findMax = require("./max.js");
+
+// Given an empty array
+// When passed to the max function
+// Then it should return -Infinity
+// Delete this test.todo and replace it with a test.
+test.todo("given an empty array, returns -Infinity");
+
+describe('max', () => {
+    test('Given an empty array', () => {
+        expect(findMax([])).toEqual(-Infinity)
+    });
+
+    test('Given an array with one number', () => {
+        expect(findMax[1]).toEqual(1)
+    });
+
+    test('Given an array with just negative numbers', () => {
+        expect(findMax[-1,-2,-3]).toEqual(-1)
+    });
+    
+    
+
+    test('Given an array with decimal numbers', () => {
+        expect(findMax[0.1,0.2,0.3]).toEqual(0.3)
+    });

```

Comment:
our test fails while trying to find out  [0.1,0.2,0.3] expected max value for this array. Can you please check your logic in the finMax function again please